### PR TITLE
Use event.target for connect/disconnect serial events

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR updates the serial article to document changes to the events fired when a serial port is connected or disconnected that are implemented in Chrome 89.
Spec: https://github.com/WICG/serial/commit/32dfe36cf081dd26f41681159a11863f5167bb91

Live preview: https://deploy-preview-4290--web-dev-staging.netlify.app/serial/#connection-disconnection

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
FYI @reillyeon

File: src/site/content/en/blog/serial/index.md [hunk 1]
```diff
@@ -393,15 +393,21 @@ access a serial port, it should monitor the `connect` and `disconnect` events.
...
-  // TODO: Remove |event.port| from the UI.
+  // TODO: Remove |event.target| from the UI.
   // If the serial port was opened, a stream error would be observed as well.
 });
 ```
 
+{% Aside %}
+Prior to Chrome 89 the `connect` and `disconnect` events fired a custom
+`SerialConnectionEvent` object with the affected `SerialPort` interface
+available as the `port` attribute.
```
Comment (reillyeon) [inline src/site/content/en/blog/serial/index.md:408]: You may want to recommend `event.port || event.target` as a way to handle the transition to the new attribute.

**Review by reillyeon — APPROVED** [review]

File: src/site/content/en/blog/serial/index.md (hunk 1 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/serial/index.md]: Done. Thanks!

**Comment by beaufortfrancois** [comment]
@jpmedley Can you review and merge?

**Review by jpmedley — APPROVED** [review]

**Pull request merged**